webapp/platform/types/src/properties.ts

```diff
@@ -31,5 +31,6 @@ export type UserPropertyFieldGroupID = 'custom_profile_attributes';
 export type UserPropertyField = PropertyField & {
     type: UserPropertyFieldType;
     group_id: UserPropertyFieldGroupID;
+    attrs?: {sort_order: number};
```

Comment:
Should `sort_order` be declared optional too?

Reply:
I was trying to strike a balance between existing fields may not have an order, but new any new changes should always have an order, but this can be added later if we need.